src/cli.rs

```diff
@@ -41,5 +60,5 @@ pub enum Command {
 #[command(author, version, about = "zkSync state reconstruction tool")]
 pub struct Args {
     #[command(subcommand)]
-    pub subcommand: Command,
+    pub subcommand: Commands,
```

Comment:
:grimacing: I guess we still need the plural? :slightly_smiling_face:

Reply:
My thinking here was that since we're using an `enum`, each variant would represent a "command" itself and not a "commands". `Command::Download` looks more *"correct"* than say `Commands::Download`.

Reply:
Ah! That makes total sense. I agree. :+1: